Source/Core/VideoCommon/FramebufferManager.h

```diff
@@ -194,7 +194,7 @@ class FramebufferManager final
   void DoLoadState(PointerWrap& p);
   void DoSaveState(PointerWrap& p);
 
-  float m_efb_scale = 0.0f;
+  float m_efb_scale = 1.0f;
```

Comment:
What does this have to do with anything?

Reply:
I read the EFB scale from the frame buffer manager in the new aspect ratio code (`g_framebuffer_manager->GetEFBScale()`).
I checked and it's always already initialized to a correct value by the time I read it, but it seemed weird to default it to 0 when the "neutral" value is 1 and 0 isn't even a valid value.